code/modules/paperwork/photography.dm

```diff
@@ -199,7 +215,13 @@
 		atoms.Add(the_turf);
 		// As well as anything that isn't invisible.
 		for(var/atom/A in the_turf)
-			if(A.invisibility) continue
+			if(A.invisibility )
+				if(see_ghosts && istype(A,/mob/dead/observer))
+					var/mob/dead/observer/O = A

```

Comment:
@RemieRichards going to sleep, but the problem we are having here is the ghost image of people on here reflects what they looked like when they died, or a tourist if they decided to observe the round. What we want to do to prevent some meta, is make it so it just looks like a generic ghost sprite. Me and Tiger are confused..and..i need sleep..going sleepy now..